doc/source/contribution/documentation/i18n.rst

```diff
@@ -66,7 +66,7 @@ Then you need to configure your cloned repository::
 
   % cd groonga
   % ./autogen.sh
-  % ./configure
+  % ./configure --enable-document
```

Comment:
すでに把握されているかもですが、ここだけmasterに直接コミットしておきました。(間違ってドキュメントの変更までmasterに入れてしまったので、そちらは取り消しておきました)

Kouhei Sutou <notifications@github.com>: 

>In doc/source/contribution/documentation/i18n.rst:
>
>> @@ -66,7 +66,7 @@ Then you need to configure your cloned repository:: > > % cd groonga > % ./autogen.sh > - % ./configure > + % ./configure --enable-document 
>
>これは別pull requestにしてもらえませんか？
>
>—
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub￼
>